packages/bat/bat.pacscript

```diff
@@ -1,25 +1,26 @@
 name="bat"
-version="0.21.0"
+version="0.22.1"
 url="https://github.com/sharkdp/bat/archive/refs/tags/v${version}.tar.gz"
-build_depends="cargo"
+build_depends="cargo libonig-dev pkg-config"
+depends="libonig5"
 replace="bat"
 description="A cat(1) clone with wings"
-hash="3dff1e52d577d0a105f4afe3fe7722a4a2b8bb2eb3e7a6a5284ac7add586a3ee"
+hash="25e45debf7c86794281d63a51564feefa96fdfdf575381e3adc5c06653ecaeca"
 maintainer="Orphaned"
-prepare() {
-        true
-}
 
 build() {
-        cargo build --release
+    RUSTONIG_SYSTEM_LIBONIG=1 cargo build -j"$(nproc)" --release --locked
```

Comment:
What's this do?

Reply:
`RUSTONIG_SYSTEM_LIBONIG` forces linking to the system libonig library, `--locked` makes cargo use dependency versions as set in the `Cargo.toml file`, this leads to better reproducible builds (with regard to upstream release)

Reply:
What's libonig?

Reply:
The oniguruma regex library that is used as a dependency by bat